NOTICE

```diff
@@ -7,3 +7,18 @@ The Apache Software Foundation (http://www.apache.org/).
 The initial code for the Polaris project was donated
 to the ASF by Snowflake Inc. (https://www.snowflake.com/) copyright 2024.
 
+--------------------------------------------------------------------------------
+
+This project includes code from Apache Iceberg, with the following
+copyright notice:
+
+| Apache Iceberg
+| Copyright 2017-2025 The Apache Software Foundation
+
+--------------------------------------------------------------------------------
+
+This project includes code from Project Nessie, developed at Dremio,
+with the following copyright notice:
+
+| Nessie
+| Copyright 2015-2025 Dremio Corporation
```

Comment:
This seems a bit too minimal. What other parts of their NOTICE files apply? As noted in the legal JIRA you pointed to the copyright lines above could be dropped.

Reply:
So, you don't need to leave in the copyright for Iceberg (as per that legal JIRA you mentioned), but you do for Nessie as it is not an ASF project.

Reply:
I did the change according to your suggestion. Does it look good to you ?